Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -458,6 +459,11 @@ void UpdateCursorSelection()
 			}
 		}
 
+		void UpdateFlowDirection()
+		{
+			(Control as UITextField).UpdateTextAlignment(Element);
```

Comment:
Does Control.UpdateFlowDirection(Element) produce the same results?

Reply:
Yes.